production/helm/loki/templates/ciliumnetworkpolicy.yaml

```diff
@@ -158,6 +158,39 @@ spec:
   {{- end }}
 {{- end }}
 
+{{- if .Values.networkPolicy.egressWorld.enabled }}
+---
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{ include "loki.name" . }}-world-egress
+  namespace: {{ .Release.Namespace }}
+spec:
+  endpointSelector:
+    matchLabels:
+      {{- include "loki.writeSelectorLabels" . | nindent 6 }}
```

Comment:
I think this is an And so this will not work and we need 3 distinct ones, or we use only selectorLabels?

Reply:
I'm not sure to understand what you mean :sweat_smile:

Reply:
I think there is no pod that will ever match backend read and write at the same time so we need 1 cnp per component or you should use {{- include "loki.selectorLabels" . | nindent 6 }}

Reply:
I updated the templating so now there are 3 different CNPs, one for each component.